ai_genomics/pipeline/gtr/make_gtr_projects.py

```diff
@@ -0,0 +1,177 @@
+# Script to explore AI definition
+
+import boto3
+import logging
+import pandas as pd
+from collections import Counter
+import re
+from toolz import pipe
+
+from ai_genomics.pipeline.gtr.gtr_utils import fetch_gtr
+from ai_genomics import config, PROJECT_DIR
+
```

Comment:
you forgot some imports -- 

```suggestion
import json
from typing import List, Dict
```